rasa_nlu_examples/tokenizers/thai_tokenizer.py

```diff
@@ -0,0 +1,39 @@
+from typing import Any, Dict, List, Text
+
+from pythainlp import word_tokenize
+
+from rasa.nlu.tokenizers.tokenizer import Token, Tokenizer
+from rasa.nlu.training_data import Message
+
+
+class ThaiTokenizer(Tokenizer):
+
+    defaults = {
+        # Flag to check whether to split intents
+        "intent_tokenization_flag": False,
+        # Symbol on which intent should be split
+        "intent_split_symbol": "_",
+        # Text will be tokenized with case sensitive as default
+        "case_sensitive": True,
```

Comment:
It seems like only `case_sensitive` is used below. Can we remove the other settings?

Reply:
I don't think so - its used by the `Tokenizer` base class from which `ThaiTokenizer` inherits.

Reply:
Ah that might be. Nevermind then :)